extensions/blocks/subscriptions/save.js

```diff
@@ -7,11 +7,11 @@ export default function Save( { attributes } ) {
 	const {
 		showSubscribersTotal,
 		submitButtonClasses,
-		customBackgroundButtonColor,
-		customTextButtonColor,
+		submitButtonCustomBackgroundColor,
+		submitButtonCustomTextColor,
 		submitButtonText,
 	} = attributes;
 	return (
-		<RawHTML>{ `[jetpack_subscription_form show_only_email_and_button="true" custom_background_button_color="${ customBackgroundButtonColor }" custom_text_button_color="${ customTextButtonColor }" submit_button_text="${ submitButtonText }" submit_button_classes="${ submitButtonClasses }" show_subscribers_total="${ showSubscribersTotal }" ]` }</RawHTML>
+		<RawHTML>{ `[jetpack_subscription_form show_only_email_and_button="true" custom_background_button_color="${ submitButtonCustomBackgroundColor }" custom_text_button_color="${ submitButtonCustomTextColor }" submit_button_text="${ submitButtonText }" submit_button_classes="${ submitButtonClasses }" show_subscribers_total="${ showSubscribersTotal }" ]` }</RawHTML>
```

Comment:
Ugh, I hadn't noticed we were saving a `shortcode` here. This should not be necessary.

Reply:
Okay. I thought it was odd.